### PR TITLE
Test dvc ls-url

### PR DESCRIPTION
**Pull request by rlamy**
Depends on iterative/dvc#8299

### CODE REVIEW

File: dvc_s3/tests/test_dvc.py [hunk 1]
```diff
@@ -45,3 +46,8 @@ def hash_value(self):
     @pytest.fixture
     def dir_hash_value(self):
         return "ec602a6ba97b2dd07bd6d2cd89674a60.dir"
+
+
+@pytest.mark.parametrize("cloud", ["s3"], indirect=True)
+class TestLsUrl(_TestLsUrl):
+    pass
```
Comment (efiop) [inline dvc_s3/tests/test_dvc.py:53]: Hm, it might be enough to just create a `cloud` fixtuer here. Similar to how we have `remote` and `workspace` above.

Comment (rlamy) [reply dvc_s3/tests/test_dvc.py:53]: Yes, that works too. I'm not sure which one is simpler though

Comment (efiop) [reply dvc_s3/tests/test_dvc.py:53]: @rlamy Let's keep it consistent for now, which means `cloud` fixture 🙂

Comment (rlamy) [reply dvc_s3/tests/test_dvc.py]: Fine by me, no strong opinion either.

**Pull request merged**